maas-region/charmcraft.yaml

```diff
@@ -130,6 +130,14 @@ parts:
 config:
   options:
     tls_mode:
+      default: "disabled"
+      description: Whether to enable TLS termination at HA Proxy ('termination'), at MAAS ('passthrough'), or no TLS ('disabled')
+      type: string
+    ssl_cert_content:
```

Comment:
what about the optional option of `ssl_cacert` in case of self-signed certificates? This could be added here and propagated accordingly to maas helper function